### PR TITLE
Upgrade templates for new definition format

### PR DESCRIPTION
**Pull request by HenningCash**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1 +1 @@
-rootProject.name = 'codefreak-templates'
+rootProject.name = 'templates'
```
Comment (erikhofer) [inline settings.gradle:1]: sure that we want to do this? implication is that, in the list of dependencies in an IDE, this shows up as a generic "templates-x.x.x.jar" 🤔

Comment (HenningCash) [reply settings.gradle:1]: I did this for local testing so the groupId and artifactId matches `com.github.codefreak:templates` but I can revert this

Comment (erikhofer) [reply settings.gradle]: Ah, that's already the artifact name because of how JitPack works. Then I think this change is actually good. re-revert? :D

Comment (HenningCash) [reply settings.gradle:1]: `get reset --hard HEAD^` :sweat_smile:

Comment (erikhofer) [reply settings.gradle:1]: grafik
:D

**Review by erikhofer — APPROVED** [review]

**Pull request merged**